sycl/plugins/cuda/pi_cuda.hpp

```diff
@@ -42,6 +42,11 @@
 #include <unordered_map>
 #include <vector>
 
+// Helper for one-liner validation
+#define PI_ASSERT(condition, error)                                            \
```

Comment:
It's a bit misleading, as it does not assert on the condition, maybe consider renaming it?

Reply:
`PI_ASSERT` to `PI_ERR_CHECK`